src/main/java/fi/aalto/cs/apluscourses/presentation/CourseProjectViewModel.java

```diff
@@ -0,0 +1,123 @@
+package fi.aalto.cs.apluscourses.presentation;
+
+import fi.aalto.cs.apluscourses.model.Course;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public class CourseProjectViewModel {
+
+  String informationText;
+  String settingsText;
+
+  private boolean cancel;
+  private boolean optOut;
+  private boolean restart;
+  private boolean restartAvailable;
+  private boolean optOutAvailable;
+
+  /**
+   * Construct a course project view model with the given course and name of the currently imported
+   * IDE settings.
+   * @param course                       The course bound to this course project view model.
+   * @param currentlyImportedIdeSettings The name of the course for which the IDE settings have
+   *                                     currently been imported.
+   */
+  public CourseProjectViewModel(@NotNull Course course,
+                                @Nullable String currentlyImportedIdeSettings) {
+    cancel = false;
+
+    String courseName = course.getName();
+    informationText = "<html><body>The currently opened project will be turned into a project for "
+        + "the course <b>" + courseName + "</b>.</body></html>";
```

Comment:
Not even nit, just a remark: At some point we could think about whether we want all UI text to be concentrated somewhere for easy access. At the same time we could consider if they should be maintained in Markdown format and only converted to HTML either in build time (may be too difficult) or run time (but in a reasonably cached way).

Reply:
Agreed, that would good. Markdown converted to HTML at compile time would be amazing, as long as we find a solution that doesn't cause IDEA's syntax highlighting to show errors at the places where the generated HTML get's inserted :smile: But even paying a small runtime cost of converting MD to HTML is worth it for the maintainability benefits of having all our UI text in one place in an easy format